jax/_src/nn/functions.py

```diff
@@ -330,7 +333,7 @@ def elu(x: ArrayLike, alpha: ArrayLike = 1.0) -> Array:
   See also:
     :func:`selu`
   """
-  numpy_util.check_arraylike("elu", x)
+  numpy_util.check_arraylike("elu", x, alpha, emit_warning=True)
```

Comment:
Here I would do
```
numpy_util.check_arraylike("elu", x)
numpy_util.check_arraylike("elu", x, alpha, emit_warning=True)
```
because we want to continue to error instead of warn if `x` is not an array